server/chat-commands/room-settings.ts

```diff
@@ -784,6 +784,7 @@ export const commands: ChatCommands = {
 	makegroupchathelp: [
 		`/makegroupchat [roomname] - Creates an invite-only group chat named [roomname].`,
 		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
+		`Requires: % @ # &`,
```

Comment:
I mean, global voices can make groupchats too.

Might want to modify it to include that I am but not sure how to do it succinctly.

Reply:
maybe it can be like this: Requires: Trusted Users

Reply:
maybe I should add it in line 785